src/apps/activities/domain/response_type_config.py

```diff
@@ -103,6 +105,18 @@ def validate_correct_answer(cls, value, values):
         return value
 
 
+class ParagraphTextConfig(_ScreenConfig, PublicModel):
+    type: Literal[ResponseType.PARAGRAPHTEXT] | None
+    max_response_length: PositiveInt = 1000
+    correct_answer: str | None = Field(
+        default=None,
+        max_length=1000,
+        description="Required if correct_answer_required is True",
+    )
+    response_required: bool
+    is_identifier: bool | None = None
```

Comment:
I don't think we need `correct_answer` or `is_identifier` for this item type based on the admin UI
![CleanShot 2024-07-24 at 09 46 56](https://github.com/user-attachments/assets/6d1214fb-1367-44bd-9e54-1cc60960d8b1)